src/gluonnlp/model/bert.py

```diff
@@ -276,11 +276,13 @@ class BERTModel(Block):
         See document of `mx.gluon.Block`.
 
     Inputs:
-        - **inputs**: input sequence tensor of shape (batch_size, seq_length)
-        - **token_types**: input token type tensor of shape (batch_size, seq_length).
+        - **inputs**: input sequence tensor, shape (batch_size, seq_length)
+        - **token_types**: input token type tensor, shape (batch_size, seq_length).
             If the inputs contain two sequences, then the token type of the first
             sequence differs from that of the second one.
-        - **valid_length**: tensor for valid length of shape (batch_size)
+        - **valid_length**: optional tensor of input sequence valid lengths, shape (batch_size)
+        - **positions**: optional tensor of position of tokens for decoding,
```

Comment:
It is hard to align `positions ` with masked positions in the sample, it would be better if we rename this to `masked_lm_token_ids`